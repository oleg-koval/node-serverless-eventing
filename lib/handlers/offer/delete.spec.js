jest.mock('../../models/loan')
const { handler } = require('./delete')

const LoanModel = require('../../models/loan')

const internal = {
    init: LoanModel,
}

beforeEach(() => {
    jest.resetAllMocks()
})

describe('delete.handler', async () => {
    it('Should remove a loan', async () => {
        internal.init.mockImplementation(async () => () => {})

        const event = {
            pathParameters: {
                id: 'b0667a9d-fa09-473e-87a5-f5753983b840',
            },
        }

        const result = await handler(event)
        expect(result).toMatchSnapshot()
    })
})
