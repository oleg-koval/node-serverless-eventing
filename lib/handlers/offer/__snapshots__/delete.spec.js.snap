// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`delete.handler Should remove a loan 1`] = `"{\\"statusCode\\":200,\\"headers\\":{}}"`;
